src/frontend/js/components/ReferralForm/ObjectField.tsx

```diff
@@ -28,11 +28,13 @@ const messages = defineMessages({
 
 interface ObjectFieldProps extends CleanAllFieldsProps {
   sendToParent: Sender<UpdateEvent>;
+  objectValue?: string;
 }
 
 export const ObjectField: React.FC<ObjectFieldProps> = ({
   cleanAllFields,
   sendToParent,
+  objectValue,
 }) => {
   const seed = useUIDSeed();
 
```

Comment:
For the initial value of regular text fields, you can just pass the initial value in the second parameter of the `useMachine` call

```
    context: {
      value: "some value"
    },
```